tools/topology/topology2/include/components/google-rtc-aec.conf

```diff
@@ -96,6 +98,7 @@ Class.Widget."google-rtc-aec" {
 			input_pin_index		0
 			in_bit_depth            16
 			in_valid_bit_depth      16
+			ibs "$[(($in_channels * ($[($in_rate + 999)] / 1000)) * ($in_bit_depth / 8)) * $M ]"
```

Comment:
@lgirdwood @singalsu @ranj063  @marcinszkudlinski , we may need unify these ibs/obs setting.
currently, when we try to set ibs/obs, we always try to do some rounding, like "+999" or "+4", based on discuss in:
https://github.com/thesofproject/sof/pull/8352/commits/77a12d548c6823407152baf36b6f638c2a61b2ee

from DP perspective, those rounding is not necessary, i, e we should use:
ibs "$[(($in_channels * ($[($in_rate] / 1000)) * ($in_bit_depth / 8)) * $M ]"

this can reduce delay for pipeline, if with rounding, take 11025 as example, it need 12 samples, if current is 11, it have to wait until next come to meet 12 samples, then if set ibs as 11 based, when input 11 samples arrived, it can go without waiting.

Reply:
Ack, thanks for the inputs, can we make a cross-tree changes with the improved formula in a seperate patch?